x-pack/plugins/case/server/routes/api/cases/comments/patch_comment.ts

```diff
@@ -61,7 +61,7 @@ export function initPatchCommentApi({ caseService, router }: RouteDeps) {
           client: context.core.savedObjects.client,
           commentId: query.id,
           updatedAttributes: {
-            ...query,
+            comment: query.comment,
```

Comment:
bug fix. the only attribute that gets updated is `comment`. The `id` field causes this error and i suspect sending `version` would as well:
```
"mapping set to strict, dynamic introduction of [id] within [cases-comments] is not allowed: [strict_dynamic_mapping_exception] mapping set to strict, dynamic introduction of [id] within [cases-comments] is not allowed"
```